packages/service-library/src/servicelib/docker_utils.py

```diff
@@ -0,0 +1,14 @@
+from datetime import datetime
+
+DOCKER_TIMESTAMP_LENGTH = len("2020-10-09T12:28:14.771034")
+
+
+def to_datetime(docker_timestamp: str) -> datetime:
```

Comment:
https://github.com/ITISFoundation/osparc-simcore/blob/fdb8a04f3afa564735b0e62da248cddb43ac02ce/services/web/server/src/simcore_service_webserver/utils.py#L55MINOR: check whether you can reuse (and therefore unify) wsome of the utils in